tests/features/joinup_search/search.feature

```diff
@@ -37,11 +37,11 @@ Feature: Global search
     # Facets should be in place.
     And the option with text "Any topic" from select facet "topic" is selected
     And the "topic" select facet should contain the following options:
-      | Any topic                   |
-      | Demography (2)              |
-      | Social and Political (2)    |
-      | Info (1)                    |
-      | Statistics and Analysis (1) |
+      | Any topic                     |
+      | Social and Political (2)      |
+      | • Demography (2)              |
+      | Info (1)                      |
+      | • Statistics and Analysis (1) |
```

Comment:
1. If I'm not missing something, the following statement is true:
   > The sum of child term counts equals the parent term count.
   
   But we need a regression test for this. Preferable to test more than one parent with many child terms.
1. We need also to add a test that asserts search result on 2-3 child terms, then the search results when filtering on parent. The test should prove that all the search entries of child terms are the same as results on parent term. Order is also important.